### DISCUSSION TITLE
Using 2 MQTT clients with for the same broker/topic within a single process

### DISCUSSION THREAD

**Discussion by YuryLapitsky-TomTom** [adam-fowler/mqtt-nio#154] [Q&A]
Hi all,

During testing I experience an issue such as that when I run XCTest with 2 MQTT-NIO clients from the same process, I don't receive updates for one of them. Unfortunately, I didn't find information about that in the documentation and code. 

So, I'd like to confirm if it might be a misconfiguration or intended behavior.

Update: 2 clients work fine when they are launched as separate processes (within iOS simulator apps)

**Comment by adam-fowler** [comment]
You should be able to use two clients at the same time as long as they have different identifiers.
